content/tutorials/getting-started/index.md

```diff
@@ -166,60 +175,113 @@ Now, you can directly skip to the [generate a package]({{< ref "#generate-packag
+```
+Do you want to create a new package or upload your project to an existing one?
+Type the number corresponding to your choice and press enter
+[0] <Quit>
+[1] Create a new package on Gatling Enterprise
+[2] Choose an existing package on Gatling Enterprise
+```
+The package is just the jar file containing your simulation class.
+Choose `1` if you don't have any package yet, to create one. It will be reusable by other simulations later on. 
 
-Under Class name, it is important you enter the name of the script that was uploaded in the package, otherwise the run of your simulation will fail in the next step.
+```
+Enter a package name, or just hit enter to accept the default name (gatling:bundle)
+Waiting for user input...
+```
 
-In this case, the ready-made script called `BasicSimulation.scala` is located in the bundle: `user-files > simulations > computerdatabase`
+Enter the name of your package, then hit **Enter**
 
-Therefore, you need to enter `computerdatabase.BasicSimulation` as the class name.
+```
+Choose the load generators location
+Type the number corresponding to your choice and press enter
+[0] <Quit>
+[1] Location AP - Hong kong
+[2] Location AP - Tokyo
+[3] Location AP Pacific - Mumbai
+[4] Location AP SouthEast - Sydney
+[5] Location Europe - Dublin
+[6] Location Europe - Paris
+[7] Location SA East - São Paulo
+[8] Location US East - N. Virginia
+[9] Location US West - N. California
+[10] Location US West - Oregon
+```
 
-Then, click on the **Next** button.
+Here we configure the Simulation itself.
+Choose the location where you want to spawn your load generators and generate traffic from.  
+For demonstration purposes we will choose the unarguable best city : `6`
 
-Next, you'll need to select your package name. It is important to select the package that contains the simulation you'd like to run. Then, click on the **Next** button.
+```
+Enter the number of load generators
+```
 
-In the final step, you can choose where you want the injection to take place, and click on the **Save** button.
+Define how many instances of load generators will be spawned in this region. Care, each load generator instance will consume credits while it runs. 
 
-Your simulation will send the traffic from the location you've selected.
+```
+Start simulation using simulation class name: computerdatabase.ComputerDatabaseSimulation
+Created simulation named gettingStartedSimulation with ID '<simulation-id>'
 
-The three next steps are optional and you can learn more about them [in the dedicated section]({{< ref "../../reference/user/simulations/" >}}), but for now, let's just simply click **Save**.
+Specify --simulation-id <simulation-id> if you want to start a simulation on Gatling Enterprise,
+or --simulation 90f60c5c-bee8-4a19-9655-866d4c49eb1d if you want to create a new simulation on Gatling Enterprise.
```

Comment:
_--simulation 90f60c5c-bee8-4a19-9655-866d4c49eb1d_ 
This is a bug fixed here: https://github.com/gatling/gatling-private/commit/b9ab9b4027067891c61ef5afffb84201de0a4d32
Replace the id by the classname